src/dns_internal.cc

```diff
@@ -128,6 +128,12 @@ class idns_query
         cbdataReferenceDone(callback_data);
     }
 
+    /// Generate 'name' from requested hostname 'orig' and configured 'searchpath'.
+    /// Subsequent calls will generate alternative FQDN from the searchpath sequence.
+    /// Skips any FQDN which exceed NS_MAXDNAME.
```

Comment:
I would drop this detail. The method may skip things for many reasons, and callers should not care about them -- it is an internal implementation detail from callers point of view. The "searchpath sequence" simply does not include FQDNs that exceed NS_MAXDNAME...


```suggestion
```